app/shared/components/Footer/footer-social-media/social-media-icon/SocialMediaIcon.tsx

```diff
@@ -12,16 +15,19 @@ interface SocialMediaIconProps {
 }
 
 const SocialMediaIcon = ({ icon, href }: SocialMediaIconProps) => {
+  const hoverStyle = socialMediaHoverMap[icon] ?? { backgroundColor: '#1A1008' };
+
   return (
-    <Link href={href} key={icon} target="_blank">
-      <IconButton>
-        <div
-          style={{
-            filter: 'brightness(0) invert(1)',
-            width: '24px',
-            height: '24px'
-          }}
-        >
+    <Link href={href} key={icon} target="_blank" sx={styles.link}>
+      <IconButton
+        sx={{
+          ...styles.iconButtonBase,
+          '&:hover': {
+            ...(hoverStyle || {})
+          }
+        }}
+      >
+        <div style={styles.iconWrapper as React.CSSProperties}>
```

Comment:
Use Box from the "@mui/material" library
```suggestion
        <Box sx={styles.iconWrapper}>
```